docs/source/algorithms/LoadILLDiffraction-v1.rst

```diff
@@ -10,15 +10,39 @@
 Description
 -----------
 
-This is the loader for the raw `.nxs` files produced by the diffractometers at ILL.
+This is the loader for the raw `.nxs` files produced by the powder diffractometers at ILL.
 Currently it supports the instruments D20 and D2B.
 
-It is able to treat the data files for arbitrary types of scans.
-The output is a point-data workspace with each spectrum corresponding to each of the detector pixels.
-The x-axis is the scanned variable axis, if there is a scan, otherwise it is empty (only one bin).
+Loading D20
+###########
 
-The exception to this is where the scanned variable is the detector, that is for detector scans on D2B.
-In this case there is only one bin, and a workspace is produced with time indexed detector information.
+For D20 1-dimensional detector, it supports 3 resolution modes:
+
+- **low** is the physical configuration, i.e. 1536 pixels. This corresponds to the IDF *D20_lr*.
+
+- **nominal** is the default configuration, i.e. pixels split by 2 by DAQ, so in total 3072 pixels. This corresponds to IDF *D20*.
+
+- **high** is when each pixel is split by 3, resulting in 4608 pixels. This corresponds to IDF *D20_hr*.
+
+Note, that all the IDFs contain only active pixels, and do not count the first and last banks went dead recently.
```

Comment:
I would say `first and last banks which are inactive.`, as recently will be misleading in a years time.